skins/oasis/modules/PageHeaderController.class.php

```diff
@@ -80,12 +80,7 @@ protected function prepareActionButton() {
 
 			$this->actionImage = MenuButtonController::ADD_ICON;
 			$this->actionName = 'addtopic';
-		} // "Edit with form" (SMW)
-		else if ( isset( $this->content_actions['formedit'] ) ) {
-			$this->action = $this->content_actions['formedit'];
-			$this->actionImage = MenuButtonController::EDIT_ICON;
-			$this->actionName = 'form-edit';
-		} // ve-edit
+		}  // ve-edit
 		else if ( isset( $this->content_actions['ve-edit'] ) && $this->content_actions['ve-edit']['main'] ) {
```

Comment:
![MINOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-minor.png) Define a constant instead of duplicating this literal "ve-edit" 10 times. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://sonarqube.wikia-inc.com/coding_rules#rule_key=php%3AS1192)